docs/index.rst

```diff
@@ -6,54 +6,76 @@ provides a simple interface for performing aperture photometry of either a singl
 image or a directory with multiple images. It is designed to be easy to use for both
 non-programmers and programmers.
 
-
-Getting Started
-===============
-
 Installation
-------------
+============
+
 
 Testing
-+++++++
+-------
 
 If you are testing a pre-release version of stellarphot we recommend setting up
 a virtual environment and installing stellarphot in this environment.
 
-Creating an environment with `conda` or `mamba`:
+Only use one of the methods below for making a virtual environment.
 
-```bash
-mamba create -n stellarphot-test python=3.11
-mamba activate stellarphot-test
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `conda` or `mamba`::
 
-Creating an environment with `virtualenv`:
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install git+https://github.com/mwcraig/stellarphot.git@update-docs
 
-```bash
-python3 -m venv stellarphot-test
-source stellarphot-test/bin/activate
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `virtualenv`::
+
+
+    python3 -m venv stellarphot-test
+    source stellarphot-test/bin/activate
+    pip install --pre --upgrade stellarphot
+
+
+Getting Started
+===============
 
 Overview
 --------
 
+You will go through this process to do photometry.
+
+#. You need to make some settings, like camera properties, observatory information, and passband maps. You may only need
```

Comment:
I made the settings file using the `1 - Savable settings.ipynb` notebook without incident EXCEPT that while I have green checkmarks for camera, observatory, and passband_map, it was saved as `partial_photometry_settings.json`.  I might suggest clearer instructions for newbies since this is a notebook and can accommodate that...

Reply:
Should the indication be that they need to continue with the notebooks? Or should the 4 notebooks be integrated into one notebook?